docs/5.0/preview/releases-table.md

```diff
@@ -1,5 +1,5 @@
-
```

Comment:
Did you make sure this table is displayed properly? I remember deliberately adding this empty line at the beginning, otherwise it would not render.

Reply:
It should work in the new docs, but lets return it here for the old ones to. To have backwards compatibility.